labextension/vpython/README.md

```diff
@@ -22,6 +22,7 @@ For a development install (requires npm version 4 or later), do the following in
 npm install
 npm install --save @jupyterlab/notebook @jupyterlab/application @jupyterlab/apputils @jupyterlab/docregistry @phosphor/disposable script-loader
 npm run build
```

Comment:
```suggestion
```

I think this can be left out too (`npm run build` seems to happen as part of `npm install`)

Reply:
Incorporated this change in squash-commit f363918.